joomla/joomla-cms
# [4.0] Field module order moved to custom elements

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/19427

### Summary of Changes
- Moving the olde code to ES6 and custom elements
- Remove chosen (=== dead code)

### Testing Instructions

Go to admin->modules, edit a module, change the position and check the order field

### Expected result



### Actual result



### Documentation Changes Required
